resources/example_map1.json

```diff
@@ -1,27 +1,28 @@
-{ "dataset":
-  {
-    "name": "MyMap1",
-    "blocks":
-    [
-      {
-      "name": "MyChr",
-      "features":
-        [
-          {
-          "name": "markerA",
-          "position": 1
-          },
-          {
-          "name": "markerB",
-          "position": 1.5
-          },
-          {
-          "name": "markerC",
-          "position": 1.8,
-          "aliases": ["markerD"]
-          }
-        ]
-      }
-    ]
-  }
+{
```

Comment:
This conversion of example_map?.json from an earlier json upload format, can be done automatically using : sed -i '/{ "dataset":/d;/^}$/d;s/"position":/"value":/;/"aliases":/d;s/^  //' example_map?.json   (based on unDataset(), posted on #dev 2018Aug07).

But further changes are required to match the current data format, and it is better to refer to the repo pretzel-data which 
contains example files based on these example_map?.json, converted into the current data format : my*.json and aliases.json.
Commit 163d3b4 drops the resources/example_map?.json and adds notes about the pretzel-data example files to the README{,.windows}.md.

To provide small example data files within this repo, we can generate from a minimal data form such as tsv using something like resources/tools/dev/effects2Dataset.pl

Reply:
Cool.  At that stage I was following the README with instructions to upload that example data, and worked out enough to see why it wasn't uploading in current form and what the data model was expecting, but hadn't seen the pretzel-data repo and didn't know the additional contextual elements now needed.  Thanks for resolving it.

Reply:
you did well to figure out the changes .. sorry I didn't anticipate that - I should have removed those files a while back, and it was only when you hit it I realised.  So it was good to get that cleaned up - don't want others to hit that pothole.   Don't hesitate to flag the errors you are getting on #dev - it can help us realise where we should improve doc or process, and often we can quickly recognise the problem.   (I've added a reference to the pretzel-data in the README)